lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -138,17 +139,19 @@ module.exports = function(
   this.openImageSelector = () => {
     analytics.logEvent('Open image selector');
 
-    OSDialogService.selectImage().then((imagePath) => {
+    osDialog.selectImage().then((imagePath) => {
+      $timeout(() => {
```

Comment:
Is this timeout needed to trigger the digest loop? If so, should we only add it around `this.selectImageByPath(imagePath)`?